app/src/main/java/com/dessalines/thumbkey/utils/KeyboardModificationService.kt

```diff
@@ -43,6 +45,7 @@ fun checkAllKeyboardModifications(
     keyModificationsErrorState.value = null
     keyModifications?.let { keyMods ->
         try {
+            if (keyModifications.isEmpty()) return
```

Comment:
I'd also recommend altering this function to require `keyModifications: String`, and doing a `!keyMods.isNullOrEmpty()` check before even getting to this function, like [here](https://github.com/dessalines/thumb-key/blob/main/app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardScreen.kt#L95).